players/installing-java/linux.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Installing Java on Linux
+description: A step by step guide on how to install Java on Linux.
+---
+
+# Installing Java on Linux
+
+This guide will walk you through installing Java 17 on Linux.
+
+The Minecraft Launcher comes with its own java installation, so this section is only relevant if you want to use the Fabric `.jar` based installer, or if you want to use the Minecraft Server `.jar`.
+
+## 1. Verify if Java is already installed.
+
+Open a terminal and type `java -version` and press <kbd>Enter</kbd>.
+
+If the command runs successfully, you will see something like this. If the command failed, proceed to the next step.
+
+![Terminal with "java -version" typed in.](/assets/players/installing-java/linux-java-version.png)
+
+::: warning
+To use the majority of modern Minecraft versions, you'll need at least Java 17 installed. If this command displays any version lower than 17, you'll need to update your existing java installation.
+:::
+
+## 2. Downloading and Installing Java 17
+
+### Arch Linux
+
+::: info
+For more information on installing Java on Arch Linux, see the [Arch Linux Wiki](https://wiki.archlinux.org/title/Java).
+:::
+
+You can install the latest JRE from the official repositories:
+
+```bash
+sudo pacman -S jre-openjdk
+```
+
+If you're running a server without the need for a graphical interface, you can install the headless version instead:
+
+```bash
+sudo pacman -S jre-openjdk-headless
+```
+
+If you plan to develop mods, you'll need the JDK instead:
+
+```bash
+sudo pacman -S jdk-openjdk
+```
+
+### Debian/Ubuntu
+
+You can install java 17 using `apt` with the following commands:
+
+```bash
+sudo apt update
+sudo apt install openjdk-17-jre
```

Comment:
I would suggest to always install the JDK, Minecraft ships with the JDK and I believe a few features (JFR) require the full JDK. Its only a little bit bigger.